packages/tools/client-debugger/client-debugger-view/src/Debugger.tsx

```diff
@@ -40,82 +54,98 @@ export interface FluidClientDebuggersProps {
  * @remarks If no debugger has been initialized, will display a note to the user and a refresh button to search again.
  */
 export function FluidClientDebuggers(props: FluidClientDebuggersProps): React.ReactElement {
-	const debuggerRegistry: DebuggerRegistry = getDebuggerRegistry();
+	const [containers, setContainers] = React.useState<ContainerMetadata[]>([]);
+	const [menuSelection, setMenuSelection] = React.useState<string>("");
```

Comment:
Can we provide a default menu selection? Not sure what we want the initial landing page to be, but it would probably be better to pick a default than require the user select one to start seeing data.

Reply:
So, the options available to us are one of the containers or the telemetry view. I don't like the telemetry view as the default... The first container in the list would be ok but we have to wait until we get the list to select it (to render it in the first place). I think a "welcome" screen of sorts might be a better option? We could put some brief descriptions of what users should expect to find under each section in the menu, or how to use the debugger (e.g. "if the container list does not show anything or seems out of date, you can click the refresh button to force a refresh").

Reply:
That's fair, and I guess it also serves to hide some of the async loading from the user, which is nice.

Reply:
And actually, it might be nice to have a landing page that points to some of our public documentation and stuff.